Gemfile

```diff
@@ -30,6 +30,8 @@ gem 'faraday'
 # background jobs using a Redis server as the queue
 gem 'sidekiq'
 
+gem 'pg'
```

Comment:
This needs to be available for all groups now that we use `foreman`, because the `worker` process will try to load `pg` as well